src/utils/util.tsx

```diff
@@ -104,3 +104,7 @@ export function useInterval(callback: () => any, delay: number, shouldPause = fa
     }
   }, [delay, shouldPause]);
 }
+
+export function smartHash(hash: string) {
+  return hash.substring(0, 8) + '...' + hash.substring(hash.length - 8)
```

Comment:
maybe we should have something like 
```
if (hash.length < 10)
   return hash
else
  return hash.substring(0, 8) + '...' + hash.substring(hash.length - 8)
```
we currently don't use this function on short hash, but nothing prevent us to do it in the future